source/process.adoc

```diff
@@ -364,3 +373,7 @@ The diagram below is a conceptual model of the transition from a Specification V
 
 .Conceptual model of the transition from a Specification Version to a Final Specification. Note that no specific packaging technology or structure should be implied from this diagram.
 image::images/image4.png[image]
+
+== Exceptions
+
+Exceptions to this process may be granted only with Super-majority approval of the Specification Committee and Executive Director. All exceptions must be documented as part of the public record.
```

Comment:
Should there be a review process for exceptions?  If exceptions are a Big Deal, we want to allow for
the greatest transparency and inclusion, right?  And shouldn't there be a requirement to document
the exception, the voting, and the results?

Reply:
> Should there be a review process for exceptions? 
> If exceptions are a Big Deal, we want to allow for the greatest transparency and inclusion, right? 

Perhaps. But exceptions are, well, exceptional. The intent is to ensure that we don't let process get in the way of doing the right thing (i.e. following the principles that underlie this process). If somebody feels that the exception process is being used unfairly, the grievance process is described in the EDP.

A Specification Committee may choose to consult with others.

> And shouldn't there be a requirement to document the exception, the voting, and the results?

The last sentence states as much. IMHO, this includes the results of votes. We've been doing this in practice. 

Do you feel that we need to explicitly state that the voting and results must be included as part of public record? If so, please propose the specific words that you'd like to see added.

Reply:
I read "the exception" as "the thing we're doing differently than what the process requires".
That doesn't include the vote.  So yes, I think you need to be explicit. Add something like:

Such documentation must include the exact change to the process, the case or conditions
under which it applies, and the votes of each Specification Committee member.

This is all still ripe for abuse by a coordinated super-majority, but I guess we'll have to
worry about that if it ever happens.

Reply:
> Such documentation must include the exact change to the process, the case or conditions under which it applies, and the votes of each Specification Committee member.

I'll add that sentence.

> This is all still ripe for abuse by a coordinated super-majority, but I guess we'll have to worry about that if it ever happens.

We ultimately have to answer to the community. We have a responsibility to operate in good faith.